Weather App/Models/Settings.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Settings.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 02.09.2021..
+//
+
+import Foundation
+
+struct Settings : Codable {
```

Comment:
Swifty way of writing code is without those spaces, after type names. Ie, `struct Settings: Codable`. Should be modified in the whole project.

Reply:
I will fix this spacings after type names.